Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesAB.swift

```diff
@@ -908,6 +945,9 @@ public struct ActorDeclSyntax: DeclSyntaxProtocol, SyntaxHashable, _LeafDeclSynt
     }
   }
   
+  /// ### Tokens
+  /// 
+  /// For syntax trees generated by the parser, this is guaranteed to be the following kind: `'actor'`
```

Comment:
@kimdv, the actor keyword doc showed up!

Reply:
I would make it `actor` and remove single quote

Reply:
We discussed that in here: https://github.com/apple/swift-syntax/pull/2168#discussion_r1319977750

But what I missed is I've put backticks back, but left single quotes in. 

To confirm, what I want to do here is: 

`/// For syntax trees generated by the parser, this is guaranteed to be of kind \`actor\`.`

Sounds good?